schematics/src/join/index.ts

```diff
@@ -20,22 +22,17 @@ function reduceTranslations(host: Tree, dirPath: string, translationJson, lang:
   if (!fs.existsSync(dirPath)) {
     throw new SchematicsException(`Could not resolve path to dir: ${dirPath}`);
   }
-  if (!hasFiles(dir)) return translationJson;
+  if (!hasFiles(dir) && !hasSubdirs(dir)) return translationJson;
   dir.subfiles
     .filter(fileName => fileName.includes(`${lang}.json`))
     .forEach(fileName => {
-      if (translationJson[key]) {

```

Comment:
this was meant to prevent overriding duplicate key... we need to keep this check